containerengine/hello-world/server.js

```diff
@@ -1,27 +0,0 @@
-// Copyright 2017 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-'use strict';
-
-// [START all]
-const http = require('http');
-const handleRequest = function (req, res) {
-  res.writeHead(200);
-  res.end('Hello Kubernetes!');
-};
-const www = http.createServer(handleRequest);
-www.listen(parseInt(process.env.PORT) || 8080);
-
-// [END all]
-module.exports = www;
```

Comment:
This is the only real "sample code" file in this `containerengine` folder, and it doesn't seem to be used anywhere.
I have checked [Google's internal Code Search](https://source.corp.google.com/search?q=containerengine%20hello-world&sq=package:piper%20file:%2F%2Fdepot%2Fgoogle3%20-file:google3%2Fexperimental).